torch/testing/_internal/common_nn.py

```diff
@@ -5471,7 +5473,7 @@ def flatten(xs):
 
 # Check that regression criterion work with no batch dimensions
 regression_criterion_no_batch = [
-    'L1Loss', 'MSELoss', 'PoissonNLLLoss', 'KLDivLoss', 'HuberLoss', 'SmoothL1Loss'
+    'L1Loss', 'MSELoss', 'PoissonNLLLoss', 'HuberLoss', 'SmoothL1Loss'
```

Comment:
We need to exclude `KLDivLoss` here, because just setting the input and target sizes (L5483-L5484) will create the inputs with `randn` (see https://github.com/pytorch/pytorch/pull/65469/files#r716708268). This would result in NaN's which would fail the gradcheck.